### PR TITLE
Breadstick

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

Adds the breadstick, which can be made with a loaf of bread and a metal rod since I didn't know what else to put as the recipe. Currently using the baguette sprite as a placeholder

Oh yeah, also a cook-specific traitor weapon for 7 tc which is visibly identical to the standard breadstick but is actually a slightly weaker captain's sabre
Sprite by TetraZeta

## Why It's Good For The Game

Listen never forget, when you're here you're family. Breadstick?

## Changelog
:cl: BootlegBow, TetraZeta
add: breadstick food
add: breadstick weapon
/:cl:

### CODE REVIEW

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Probably should add an icon for breadstick and tweak sabre item, 10 TC for stealthy 10 force weapons is too much.

File: ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm [hunk 1]
```diff
@@ -157,6 +157,15 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 10
 
+/obj/item/melee/sabre/breadstick
+	name = "breadstick"
+	desc = "Listen never forget, when you’re here you’re family. Breadstick?"
+	icon_state = "breadstick"
+	icon = 'ModularTegustation/Teguicons/breadstick.dmi'
+	inhand_icon_state = "baguette"
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm:165]: You have to add an inhand_icon file.

Comment (BootlegBow) [reply ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm:165]: alright

File: ModularTegustation/tegufood.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	result = /obj/item/food/bread/breadstick
+	subcategory = CAT_BREAD
+
+//Actual objects
+
+/obj/item/food/bread/breadstick
+	name = "breadstick"
+	desc = "Listen never forget, when you're here you're family. Breadstick?"
+	icon_state = "baguette"
+	icon = 'icons/obj/food/burgerbread.dmi'
```
Comment (EgorDinamit) [inline ModularTegustation/tegufood.dm:18]: Same here, + you forgot to change icon.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
So yeah, fix the icons and it's probably good to go.

**Comment by BootlegBow** [comment]
@EgorDinamit it seems there actually is no inhand sprite for the baguette, and if there is I can't find it. Unless you want to wait for an inhand sprite specially for the breadstick, I think this is good to go

File: ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm [hunk 3]
```diff
@@ -157,6 +157,14 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 10
 
+/obj/item/melee/sabre/breadstick
```
Comment (JohnFulpWillard) [inline ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm:160]: Why is this put right stack in the middle of the traitor viro kit code?

Comment (BootlegBow) [reply ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm:160]: That would be my mistake in not reading enough of the code around it.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Right now it will use normal sabre's inhand icon, I think you should replace it...

**Comment by BootlegBow** [comment]
> Right now it will use normal sabre's inhand icon, I think you should replace it...

We'll need a special sprite for it then, which is... difficult

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**